src/main/java/net/pms/database/DatabaseHelper.java

```diff
@@ -207,44 +216,58 @@ public abstract class DatabaseHelper {
 	protected static final String FALSE = "FALSE";
 	protected static final String NULL = "NULL";
 	protected static final String TRUE = "TRUE";
+	protected static final String EMPTY_STRING = "''";
 	protected static final String PARAMETER = "?";
+	protected static final String STRINGENCODE_PARAMETER = "STRINGENCODE(" + PARAMETER + ")";
+	protected static final String LIKE_STARTING_WITH_PARAMETER = STRINGENCODE_PARAMETER + " || '%'";
+	protected static final String LIKE_ENDING_WITH_PARAMETER = "'%' || " + STRINGENCODE_PARAMETER;
+	protected static final String LIKE_CONTAIN_PARAMETER = "'%' || " + LIKE_STARTING_WITH_PARAMETER;
 
 	/**
 	 * SQL COMMANDS
 	 */
 	protected static final String CREATE = "CREATE ";
-	protected static final String INSERT = "INSERT ";
+	protected static final String INSERT_INTO = "INSERT INTO ";
 	protected static final String SELECT = "SELECT ";
 	protected static final String UPDATE = "UPDATE ";
+	protected static final String WITH = "WITH ";
 
 	protected static final String ADD = " ADD ";
 	protected static final String DROP = " DROP ";
 	protected static final String ALTER = "ALTER ";
 	protected static final String AND = " AND ";
+	protected static final String AS = " AS ";
 	protected static final String ASC = " ASC";
-
 	protected static final String COLUMN = "COLUMN ";
 	protected static final String CONSTRAINT = "CONSTRAINT ";
 	protected static final String DELETE_FROM = "DELETE FROM ";
+	protected static final String DESC = " DESC";
 	protected static final String EQUAL = " = ";
+	protected static final String EQUAL_0 = EQUAL + "0";
 	protected static final String EXISTS = "EXISTS ";
 	protected static final String FROM = " FROM ";
+	protected static final String GREATER_OR_EQUAL_THAN = " >= ";
+	protected static final String GREATER_THAN = " > ";
 	protected static final String IF = "IF ";
+	protected static final String IN = " IN ";
+	protected static final String IS = " IS ";
 	protected static final String INDEX = "INDEX ";
+	protected static final String JOIN = " JOIN ";
```

Comment:
I think we could rethink this fine-grained approach to query building, it makes code less readable and discoverable. Is this abstraction done for performance? Is it significant?